internal/executor/configuration/types.go

```diff
@@ -29,6 +29,8 @@ type TaskConfiguration struct {
 	PodDeletionInterval                   time.Duration
 	PodMetricsInterval                    time.Duration
 	QueueUsageDataRefreshInterval         time.Duration
+	UtilisationEventReportingInterval     time.Duration
+	UtilisationEventReportingPeriod       time.Duration
```

Comment:
Not totally against this. However everything else in this struct is a task interval. This isn't a task interval.